app/views/pages/_search_result.html.erb

```diff
@@ -22,14 +22,9 @@
           <strong><%= Category.model_name.human %>:</strong>
           <%= item.category.description %>
         </p>
-        <%# TODO: refactoring %>
-        <% if item.in_page? @page %>
-          <span class="already"><%= t 'views.item_already_in_this_page' %></span>
-        <% else @page %>
-          <%= button_to({action: 'add_to_page'}, remote: true,
-                params: {item: {id: item.id}}, class: 'button-box') do %>
-            <%= t 'views.add_to_page' %>
-          <% end %>
+        <%= button_to({action: 'add_to_page'}, remote: true,
```

Comment:
Does it make sense to leave/add `<% if @page %>` before adding the button tag?

Reply:
I'm not sure, I'm trying to imagine a scenario where it may break if @page isn't present.